proto/streamlit/proto/AppPage.proto

```diff
@@ -29,4 +29,9 @@ message AppPage {
   string page_script_hash = 1;
   string page_name = 2;
   string icon = 3;
+
+  // A feature for MPA v2 and inform the frontend what's the default page
```

Comment:
nit: I think it should be "A feature for MPA v2 and inform" -> "A feature for MPA v2 to inform"